src/datasources/transaction-api/transaction-api.service.ts

```diff
@@ -436,4 +436,13 @@ export class TransactionApi implements ITransactionApi {
       throw this.httpErrorFactory.from(error);
     }
   }
+
+  async deleteDeviceRegistration(uuid: string, safeAddress: string) {
```

Comment:
This is returning a `NetworkResponse` to the domain layer right?

I'd not expose this to the domain/route layers.